api/openvpn-rw/connection-history-csv

```diff
@@ -22,18 +22,31 @@
 
 import sqlite3
 import datetime
+import sys
+import os
+import syslog
 from pytz import timezone
 
-conn = sqlite3.connect('/var/lib/nethserver/openvpn/connections.db')
-c = conn.cursor()
-connections = []
+DATABASE_PATH = '/var/lib/nethserver/openvpn/connections.db'
+database_exists = os.path.isfile(DATABASE_PATH)
 
-rows = c.execute('''SELECT virtual_ip_addr, remote_ip_addr, start_time, duration, bytes_received, bytes_sent, common_name
-                    FROM connections
-                    ORDER BY start_time DESC''')
+if not database_exists:
+    syslog.syslog(syslog.LOG_ERR, 'Error: {} does not exist. Please execute /etc/e-smith/events/actions/nethserver-openvpn-create-connections-db'.format(DATABASE_PATH))
+    sys.exit(1)
```

Comment:
Write directly to stderr, make sure that the caller script doesn't redirect stderr to /dev/null